tests/test_expertise_service.py

```diff
@@ -0,0 +1,451 @@
+                    'name': 'test_run',
+                    'match_group': ["ABC.cc"],
+                    'paper_invitation': 'ABC.cc/-/Submission',
+                    "model": "elmo",
+                    "model_params": {
+                        'dummy_param': '64',
+                        "use_title": False,
+                        "use_abstract": True,
+                        "average_score": True,
+                        "max_score": False
+                    }
+                }
+            ),
+            content_type='application/json'
+        )
+        assert response.status_code == 400, f'{response.json}'
+        assert 'bad request' in response.json['error'].lower()
+        assert response.json['error'] == 'Bad request: unexpected model param: dummy_param'
+
+    def test_request_expertise_with_valid_parameters(self, openreview_context, celery_session_app, celery_session_worker):
+        # Submit a working job and return the job ID
+        test_client = openreview_context['test_client']
+        # Make a request
+        response = test_client.post(
+            '/expertise',
+            data = json.dumps({
+                    'name': 'test_run',
+                    'match_group': ["ABC.cc"],
+                    'paper_invitation': 'ABC.cc/-/Submission',
+                    "model": "elmo",
+                    "model_params": {
+                        "use_title": False,
+                        "use_abstract": True,
+                        "average_score": True,
+                        "max_score": False
+                    }
+                }
+            ),
+            content_type='application/json'
+        )
+        assert response.status_code == 200, f'{response.json}'
+        job_id = response.json['job_id']
+
+        response = test_client.get('/expertise/status', query_string={'id': f'{job_id}'}).json['results']
+        assert len(response) == 1
+        assert response[0]['name'] == 'test_run'
+        assert response[0]['status'] != 'Error'
+        # assert response[0]['description'] == 'Server received config and allocated space'
+
+        # # Attempt getting results of an incomplete job
+        # time.sleep(5)
+        # response = test_client.get('/expertise/results', query_string={'id': f'{job_id}'})
+        # assert response.status_code == 500
+
+        # Check for queued status
+        #time.sleep(5)
+
+        # response = test_client.get('/expertise/status', query_string={'id': f'{job_id}'}).json['results']
+        # assert len(response) == 1
+        # assert response[0]['name'] == 'test_run'
+        # assert response[0]['status'] == 'Queued'
+        # assert response[0]['description'] == 'Server received config and allocated space'
+
+        # Query until job is complete
+        response = test_client.get('/expertise/status', query_string={'id': f'{job_id}'}).json['results']
+        assert len(response) == 1
+        while response[0]['status'] != 'Completed':
```

Comment:
This can be an infinite loop. Fix the number of tries and mark the test as failure after that.